src/Widgets/ControlPanel.hs

```diff
@@ -65,6 +75,10 @@ controlPanel = mdo
                 .#{cstate}
                     box-shadow: 0
                     right: -400px
+
+            .tab-content
```

Comment:
Also, try to avoid literal names for css classes. Try to use unique names generated by function newVar - this way we avoid accidental css classname clashes.